base/abstractarraymath.jl

```diff
@@ -92,6 +92,10 @@ imag{T<:Real}(x::AbstractArray{T}) = zero(x)
 +{T<:Number}(x::AbstractArray{T}) = x
 *{T<:Number}(x::AbstractArray{T,2}) = x
 
+function lerpi(j::Integer, d::Integer, A::AbstractArray, B::AbstractArray)
```

Comment:
Hmm, now I'm getting different performance results. Maybe I screwed up somewhere, but it's something to keep an eye on (possibly a function-redefinition issue?).